dapps/marketplace/translation/README.md

```diff
@@ -32,9 +32,9 @@ For tag properties, e.g. `tooltip`, use the `fbt()` javascript function. Like th
 	  placement="bottom"
 	>
 
-It is important to wrap _entire sentances_ that can be translated. That is, do not break a sentance across multiple `<fbt>` tags. This is important because words go in different orders in different languages, and in any case the translators will not know which fragments are meant to be together. 
+It is important to wrap _entire sentances_ that can be translated. That is, do not break a sentance across multiple `<fbt>` tags. This is important because words go in different orders in different languages, and in any case the translators will not know which fragments are meant to be together.
```

Comment:
the french guy picking on the native speaker.... isn't it sentence rather than sentance ?